registry.json

```diff
@@ -5655,5 +5655,20 @@
     "telegram": "BenjaminAnvil",
     "operatorName": "",
     "github": "Cardano-Forge"
-  }
+  },
+  {
+    "ticker": "PFL01",
+    "poolId": "24bfb62716d755bbf035fcb3f7e87a8d325fde0f04062e692aff3216",
+    "social": {
+      "website": "https://poolforlovelace.me",
+      "twitter": "PFL_pool",
+      "facebook": "",
+      "discord": "",
+      "instagram": "",
+      "youtube": ""
+    },
+    "telegram": "PFL01",
```

Comment:
Please can you add your telegram handle here? You can find it by opening your settings and it should appear under your display name, e.g. @poolforlovelace

Reply:
Hi I updated name telegram  =>   "telegram": "pfl_01"

Cordially